regulations/tasks.py

```diff
@@ -31,6 +38,8 @@ def submit_comment(body):
                 'X-Api-Key': settings.REGS_API_KEY,
             }
         )
+        response.raise_for_status()
+        logger.info(response.text)
```

Comment:
Quick check -- is there anything in this response that could be considered sensitive, or is it just the submission id, or similar?